x/bank/legacy/v0_39/migrate.go

```diff
@@ -0,0 +1,22 @@
+package v039
+
+import (
+	v038auth "github.com/cosmos/cosmos-sdk/x/auth/legacy/v0_38"
+	v038bank "github.com/cosmos/cosmos-sdk/x/bank/legacy/v0_38"
+)
+
+// Migrate accepts exported x/auth and x/bank genesis state from v0.38 and migrates
+// it to v0.39 x/bank genesis state. The migration includes:
+//
+// - Moving balances from x/auth to x/bank genesis state.
+func Migrate(bankGenState v038bank.GenesisState, authGenState v038auth.GenesisState) GenesisState {
```

Comment:
Do we need to ensure that this `Migrate` runs before `auth.Migrate`? Is there anything that needs to be done to ensure that?